### DISCUSSION TITLE
how to install lightroom presets

### DISCUSSION THREAD

**Discussion by ERNESTDL** [rsjaffe/MIDI2LR#658] [Q&A] [answered]
I just installed MIDI2LR on a MIDI KORG NanoKontrol 2

I use lightroom classic 11.3.1

I would like to know how to assign to MIDI the presets that I normally use, I can't find the way to do it, does anyone have a link or a tutorial to do it?

Thanks

**Comment by rsjaffe** [comment, marked as answer]
Here are the instructions: https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Develop-Presets-Dialog

**Reply by ERNESTDL** [reply]
Thanks!!!!!
